packages/valory/skills/termination_abci/behaviours.py

```diff
@@ -402,10 +403,10 @@ def _get_multisend_tx(self) -> Generator[None, None, Optional[str]]:
         # we remove all but one safe owner
         for owner in safe_owners[1:]:
```

Comment:
I did not study the logic here. Is the service owner always the first of the safe owners? Or can there be a situation where the first is not, and the first needs to transfer control to the owner. This would mean there is a moment that a single agent - operated by someone else than the service owner - is in control of the vault.

Reply:
No, so what happens is we remove all owners but one, and then the last owner that is left we swap with the service owner.